### ISSUE TITLE
For a multiclass classification problem, how the raw outputs of predict_proba are mapped into the probability space?

### ISSUE THREAD

**Issue by marcosbd** [microsoft/LightGBM#5092]
That is, how the output of `.predict_proba(X, raw_output = True)` are mapped into the outputs of `.predict_proba(X)`?

**Comment by Jakobhenningjensen** [comment]
I'm pretty sure it's just the SoftMax function of the outputs but I might be wrong

**Comment by jmoralez** [comment]
As @Jakobhenningjensen said, the softmax is applied. The process first computes the raw scores:
https://github.com/microsoft/LightGBM/blob/3ed0027bffe820ba6c81f32b82f02ef8302c6aad/src/boosting/gbdt_prediction.cpp#L55-L65
and in the case of multi-class classification the `ConvertOutput` method is the softmax:
https://github.com/microsoft/LightGBM/blob/3ed0027bffe820ba6c81f32b82f02ef8302c6aad/src/objective/multiclass_objective.hpp#L132-L134
